composer.json

```diff
@@ -18,7 +18,11 @@
     "require-dev": {
         "cakephp/cakephp": "dev-3.next as 3.4",
         "cakephp/cakephp-codesniffer": "^2.1",
-        "phpunit/phpunit": "@stable"
+        "phpunit/phpunit": "@stable",
+        "firebase/php-jwt": "~4.0"
+    },
+    "suggest": {
+        "firebase/php-jwt": "~4.0"
```

Comment:
Yup, the right side should be something like `If you want to use the 'JwtAuthenticator'`.